docs/D-kit/Vehicle_Guide/Apollo_D-kit_sensor_integration_V04.md

```diff
@@ -0,0 +1,91 @@
+# **Apollo传感器集成说明**
+
+传感器上车集成前，请优先或同步完成工控机的Apollo系统安装，教程详见“循迹搭建——Apollo系统安装”。工控机安装Apollo系统时开采用自带的直流电源模块供电，显示器可使用电脑显示器。
+
+## **快速介绍**
+
+请您在传感器集成前仔细阅读本文档。本文档介绍了传感器供电线束安装的注意事项。
+
+## **Apollo上装电气原理图**
+
+Apollo上装电气原理图如下图所示：
+![Apollo_electrical_schematic](./images/Apollo_electrical_schematic.png)
+通过电气原理图可知，
+通过电气原理图可知，6108/8108工控机（IPC）由底盘独立提供24V电源，显示器、激光雷达、毫米波雷达、IMU和路由器则由底盘通过保险盒提供12V电源。
+
+## **1. 上装传感器供电线束简介**
+
+Apollo电气原理图中的供电部分集成在了一条供电线束中，如下图所示。
+![apollo_power_wire_harness](./images/apollo_power_wire_harness.png)
+
+保险丝盒取消了负极接线，负极回路集成在线束内部。
+
+底盘后部电气面板如下图所示：
+![Electrical_panel](./images/Motor_Enable_1.png)
+上装供电插座的脚位定义如图所示:1、Pin1—12V+；2、Pin2—12V-；3、Pin3—24V+；4、Pin4—24V-。
+
+![apollo_to_chassis](./images/apollo_to_chassis.gif)
+底盘can口通过db9延长线与工控机相连。上装供电线通过4pin航插与底盘供电插座相连。
+**<font color=#FF0000 >提示：请在检查供电线束与传感器连接插头极性之后，再连接供电线束与底盘供电口的航插。</font>**
+
+## **2. 传感器电源接口介绍**
+
+1、工控机电源接口
+![8108_Power_1](./images/8108_Power_1.png)
+![8108_Power_2](./images/8108_Power_2.png)
+![8108_Power_3](./images/8108_Power_3.png)
+2、M2电源接口
+M2自带数据线缆为1分5线缆，一端为与工控机相连的航插接头，另一端为M2电源输入口，网口、激光雷达授时输出口，USB输出口，db9输出口（M2配置串口）。详细如下图所示：
+![M2_1](./images/M2_1.png)
+
+注意：M2线缆电源输入口为白色2pin插头，红色为12V正极，蓝色为12V负极；上装线束M2供电插头为蓝色2pin插头，红色为12V正极，黑色色为12V负极，对插后红色对红色，黑色对蓝色。连接插头时请确认线色或用万用表检查线束端供电插头电源极性。连接如下图所示：
+![M2_2](./images/M2_2.png)

```

Comment:
The picture has the same discription with above words, modified the picture please.